specs/order_spec.rb

```diff
@@ -76,34 +76,66 @@
       result.must_equal true
     end
   end
+
+  describe "remove_product" do
+    #checks if number of products is one less
+    it "checks if removes product" do
+      products = {"frozen pizza" => 6.50, "cat food" => 7.00, "tissues" => 3.25, "The Game by Neil Strauss" => 12.50, "Best Brand lotion" => 2.00, "laundry detergent" => 8.50}
+      order = Grocery::Order.new(1337, products)
+      before_count = products.count
+      order.remove_product("laundry detergent")
+      expected_count = before_count - 1
+      order.products.count.must_equal expected_count
+    end
+    it "removes the correct item"
+    it "checks if removes product" do
+      products = {"frozen pizza" => 6.50, "cat food" => 7.00, "tissues" => 3.25, "The Game by Neil Strauss" => 12.50, "Best Brand lotion" => 2.00, "laundry detergent" => 8.50}
+      order = Grocery::Order.new(1337, products)
+      order.remove_product("laundry detergent")
+      order.products.keys.wont_include("laundry detergent")
+    end
```

Comment:
I like these tests. Here's another one to think about: what happens if you try to remove a product that isn't part of the order?